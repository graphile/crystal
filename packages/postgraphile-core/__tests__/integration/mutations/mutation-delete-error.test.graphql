mutation {
  deleteTypeById(input: { id: 11 }) {
    clientMutationId
    deletedTypeId
    type {
      nodeId
      id
    }
    query {
      nodeId
    }
  }
}
