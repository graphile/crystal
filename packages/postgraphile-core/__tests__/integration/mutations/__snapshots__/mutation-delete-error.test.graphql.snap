// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "deleteTypeById": null,
  },
  "errors": Array [
    [GraphQLError: Nope.],
  ],
}
`;
